### DISCUSSION TITLE
Возможности платформы для запуска и не только

### DISCUSSION THREAD

**Discussion by pepelsbey** [doka-guide/platform#230] [Разное]
Мы хотим понять, какие возможности платформы нам нужны для запуска и сразу после, чтобы собрать майлстоун и расставить там приоритеты. Давайте накидаем сюда всё, что придёт в голову. Предлагаю такой сценарий:

1. Откройте [y-doka.site](https://y-doka.site/)
2. Посмотрите, что там есть и главное — чего там нет
3. Подумайте, что вы хотели бы видеть

К возможностям платформы относится всё, что мы сможем сделать силами Eleventy и дизайнеров, чтобы вывести данные из контента максимально выгодным и удобным образом — как для чтения, так и для создания и правок на Гитхабе.

**Comment by lenaryan** [comment]
Тёмная тема? А то всё прям очень белое :)

**Reply by pepelsbey** [reply]
Да, это прямо тренди. У дизайнеров есть запрос на тёмную тему и её будут рисовать.

**Reply by lenaryan** [reply]
У меня же Дима Волжский дривен девелопмент, кек)))

**Comment by bespoyasov** [comment]
**Очень не хватает тематических подборок**

Сейчас есть только поиск и список всех статей. Хочется выдавать пользователям порционный-тематически-связанный контент в виде нескольких ссылок по какой-то теме.

**Тизеры статей**

Возможно, для вовлечения стоит на главную (?) выводить тизеры статей. Либо самых залайканных, либо самых свежих, либо черновиков-заглушек, которые пользователи могут написать сами.

**Reply by igsekor** [reply]
Можно попробовать сформировать граф связей статей. Что скажите?

**Reply by bespoyasov** [reply]
Knowledge base 😎

**Comment by igsekor** [comment]
1. Поиск со всем набором функционала (Algolia -> своё решение)
2. Линтеры для кода / валидаторы для HTML
3. Unit-тесты для кода / Unit-тесты для демок
4. UI-тесты
5. Зависимые экшены для оптимизации затраченного времени на проверку
6. Docker-образ для быстрого развёртывания веб-сервера с сайтом на любом сервере
7. Docker-образ для развёртывания CDN (на будущее)
8. Автоматические скриншоты в статьях о сервисах

**Comment by furtivite** [comment]
1. Тестирование знаний после прочтения и геймификация
1. Личный кабинет с закладками
2. Версионирование статей и уведомление об изменениях

**Comment by pepelsbey** [comment]
- RSS-фид с новыми статьями
- Картинка для соцсетей
  - 1 шаг: дефолтная для категории в её цветах
  - 2 шаг: генерируемая из заголовка статьи
- Генерация версий картинок из маркдауна

**Reply by furtivite** [reply]
> RSS-фид с новыми статьями

И несколько подфидов: только с энциклопедийными статьями (ака дока), только со статьями (экс. лонгридами)
